### DISCUSSION TITLE
빌드잇 오브젝트 collider 에러

### DISCUSSION THREAD

**Discussion by hipsc159** [naverz/zepeto-studio-kor#360] [5. Q&A]
빌드잇에 있는 오브젝트를 사용 중인데, 어떤 것은 coliider가 적용이 되고 어떤 것은 안되네요.

오브젝트 마다 mesh collider를 적용했는데, 같은 문제가 발생합니다.

![KakaoTalk_20220210_160737482](https://user-images.githubusercontent.com/68143858/153356090-e3a9803b-d554-4b72-958b-163111e834e4.png)

빌드잇에서 오브젝트 가져오신 분들 어떻게 해결하셨는지 알 수 있을까요??

**Comment by runasy-koonta** [comment]
Mesh Collider의 Mesh가 오브젝트의 모양에 맞추어 제대로 적용 되어 있는 상태인데 collider가 작동하지 않는 것인가요?
만약 맞다면, 폴리곤 255개 제한에 걸렸을 가능성도 있어 보입니다.

**Reply by hipsc159** [reply]
넵 감사합니다!
